src/model/debtItem.ts

```diff
@@ -3,10 +3,15 @@ export default class DebtItem {
     category: string;
     comment: string;
     fileName: string;
-    constructor(type: string, category: string, comment: string, fileName:string) {
+    price?: number;
+
+    constructor(type: string, category: string, comment: string, fileName:string, price?:number) {
         this.type = type;
         this.category = category;
         this.comment = comment;
         this.fileName = fileName;
+        if (price) {
+            this.price = price;
```

Comment:
tu peux laisser this.price = price non ?

Reply:
En effet, bien vu !